starter-code/app.js

```diff
@@ -1,5 +1,92 @@
 const express = require('express');
 const app = express();
+const layouts      = require('express-ejs-layouts');
```

Comment:
I think you forgot to do `--save` when you did `npm install express-ejs-layouts`  The difference is that when I download the file on my computer, when I do `npm install` it doesn't get installed so I have to install it separately.  You can tell because it's not in the dependencies in the package.json file.